tests/helpers/stubs.py

```diff
@@ -266,6 +266,9 @@ def load_status(self):
     def shutdown(self):
         pass
 
+    def icon(self):
+        return self.windowIcon()
```

Comment:
I think it might be a bit cleaner to return a plain `QIcon()` here, but I don't think that matters too much.

Reply:
I was thinking about returning `QIcon()`, but went with `windowIcon()` because it doesn't need a extra import line. Easy to change, though.

Reply:
I also prefer `QIcon()`, but I fixed that up in 32568a6da44468a03a2e08b1a0cbe8c4a913c3ca